src/specialOffer/SpecialOffer.story.mdx

```diff
@@ -0,0 +1,198 @@
+import { action } from '@storybook/addon-actions'
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks'
+
+import { ContentDivider } from '../divider/contentDivider'
+import { ItemsList } from '../itemsList'
+import { BaseSection } from '../layout/section/baseSection'
+import { SpecialOffer } from './index'
+
+<Meta title="Widgets/SpecialOffer" />
+
+# **SpecialOffer**
+
+<Canvas>
+  <Story name="Default">
+    <BaseSection>
+      <ItemsList>
+        <SpecialOffer
+          picture={{
+            url: 'https://cdn.blablacar.com/kairos/assets/build/images/blablasure_logo-459e4b952a6cc04b72ea0e12f98ec673.svg',
+            alt: 'The BlaBlaSur offer logo',
+          }}
+          title="A wonderful offer"
+          description="Once you receive your first rating, receive a 30€ voucher on Total spring green electricity offer for your home. So come back here to enjoy it !"
+          informationLink={{
+            label: 'Learn more about the conditions',
+            url: 'http://example.com/learn-more',
+          }}
+          ctaLinks={[
+            {
+              label: 'Register for option 1!',
+              url: 'http://example.com/register1',
+            },
+            {
+              label: 'Register for option 2!',
+              url: 'http://example.com/register2',
+            },
+          ]}
+        />
+        <ContentDivider />
```

Comment:
Would be better to split into 2 Stories if you want to show different cases

Reply:
Okey, I will remove `BaseSection` to have only `SpecialOffer` here